### ISSUE TITLE
Revamp the current startup screen

### ISSUE THREAD

**Issue by AlexisDrogoul** [project-SIMPLE/GamaServerMiddleware#13]
The current startup screen, already well simplified, has still useless information, a somewhat inconsistent design and lacks important information.

Experiment buttons: "Select a simulation" should not be necessary as the buttons should be self explanatory. The buttons are too big, especially for when we have more experiments available. They should be somewhere, and somewhat not so different than "apps" on an iPhone for instance. 

Enter project directory path: this is so prominent on the page that someone would think that it is necessary to enter something in order to run the examples. It should be hidden behind a "Custom..." button equivalent to the other ones, and open a custom page (or, better, unveil the text input and control to choose the path). "Launch selected simulation" has a pause icon, which is completely counter-intuitive. 

Get simulations informations: title must be changed to something shorter and informative ("Status" ? ), and it should not be positioned like this in the middle of nowhere, immediately attracting the attention while it shouldnt. Why is there also a 'pause' icon on this button ? this is misleading. 

There should be a footer that points to the project website, and indicates that it is funded by EU Global Gateway, designed by IRD. 

So, in a nutshell, I would expect a design close to the mockup below ! 

![Untitled 3](https://github.com/user-attachments/assets/3dc27188-3ad8-485b-9f8c-ea5d514c455e)
